torch/_torch_docs.py

```diff
@@ -595,10 +595,10 @@ def merge_dicts(*dicts):
            r"""
 atan2(input, other, out=None) -> Tensor
 
-Returns a new tensor with the arctangent of the elements of :attr:`input`
-and :attr:`other`.
+Returns a new tensor with the arctangent of the angle between
```

Comment:
It's not really the arctangent of the angle here? The angle is the arctangent no?

Reply:
You are right, it should be the angle